code/modules/research/techweb/all_nodes.dm

```diff
@@ -597,11 +597,11 @@
 
 /datum/techweb_node/adv_rcd_upgrade
 	id = "adv_rcd_upgrade"
-	display_name = "Advanced RCD designs upgrade"
+	display_name = "Advanced Bluespace RCD designs."
```

Comment:
```suggestion
	display_name = "Bluespace-Integrated Rapid Construction"
```
Sounds more flashy. At the very least, don't include a period at the end, since no other `display_name` has one.